content/cdi-i/0014-series-numericas.md

```diff
@@ -0,0 +1,1195 @@
+
+$$
+\sum^\infin_{n=p}\left(K\cdot r^n\right)
+$$
+
+com $K,r\in\R$.
+
+Uma série geométrica é convergente se e só se $|r|<1$ e, nesse caso, a soma da série dada é:
+
+$$
+S=\frac{K\cdot r^p}{1-r}
+$$
+
+:::details[Exemplos]
+
+$$
+\sum^\infin_{n=1}2^n
+$$
+
+É [**divergente**](color:pink), pois $|r|>1$.
+
+---
+
+$$
+\sum^\infin_{n=1}2\times3^{-n}=\sum^\infin_{n=1}2\times\left(\frac{1}{3}\right)^{n}
+$$
+
+É [**convergente**](color:green), pois $|r|<1$, e a sua soma é $S=\frac{2\cdot \left(\frac 13\right)^1}{1-\frac13}=\frac23\times\frac32=1$.
+
+:::
+
+## Série de Mengoli
+
+Uma série de Mengoli é uma série que tem uma das duas formas:
+
+$$
+\sum ^{\infty }_{n=p}( u_{n} -u_{n+1}) \quad \text{ou} \quad \sum ^{\infty }_{n=p}( u_{n+1} -u_{n})
+$$
+
+e é convergente se e só se $(u_n)$ é uma sucessão convergente.
+
+Nesse caso, a sua soma é [$S=u_p-\lim u_n$](color:orange) para o primeiro caso e [$S=\lim u_n-u_p$](color:orange) para o segundo.
+
+:::details[Exemplos]
+
+$$
+\sum ^{\infty }_{n=1}\left(\frac{1}{n+5} -\frac{1}{n+4}\right)
+$$
+
+Logo, é uma série de Mengoli da forma
+
+$$
+\sum ^{\infty }_{n=1}( u_{n+1} -u_{n}) \quad \text{com} \quad u_{n} =\frac{1}{n+4}
+$$
+
+Como $\lim u_n=0\in\R$, a série é [**convergente**](color:green) e a sua soma é:
+
+$$
+u_{1} =\frac{1}{1+4} =\frac{1}{5} \quad \quad S=0-\frac{1}{5} =-\frac{1}{5}
+$$
+
+---
+
+$$
+\sum ^{\infty }_{n=0}\left(\frac{1}{1+e^{-n}} -\frac{1}{1+e^{-n-1}}\right)
+$$
+
+Logo, é uma série de Mengoli da forma
+
+$$
+\sum ^{\infty }_{n=0}( u_{n} -u_{n+1}) \quad \text{com} \quad u_{n} =\frac{1}{1+e^{-n}}
+$$
+
+Como $\lim u_n=1\in\R$, a série é [**convergente**](color:green) e a sua soma é:
+
+$$
+u_{0} =\frac{1}{1+e^{0}} =\frac{1}{2} \quad \quad S=\frac{1}{2} -1=-\frac{1}{2}
+$$
+
+---
+
+$$
+\sum ^{\infty }_{n=1}\left(\sin\frac{n\pi }{4} -\sin\frac{( n+1) \pi }{4}\right)
+$$
+
+Também é uma série de Mengoli da forma
+
+$$
+\sum ^{\infty }_{n=1}( u_{n} -u_{n+1}) \quad \text{com} \quad u_{n} =\sin\frac{n\pi }{4}
+$$
+
+No entanto, como a sucessão $(u_n)$ é divergente, a série é [**divergente**](color:pink).
+
+:::
+
+## Série Redutível
+
+Diz-se que uma série é redutível se existem $\alpha\in\R\backslash\{0\}$ e $k,p\in\N^+$ tais que a série se pode escrever na forma
+
+$$
+\sum ^{\infty }_{n=p}\left[ \alpha ( u_{n} -u_{n+k})\right]
+$$
+
+A série é convergente se e só se a sucessão definida por
+
+$$
+v_n=\sum^{k-1}_{j=0}u_{n+j}
+$$
+
+é uma sucessão convergente. Nesse caso, a soma da série é dada por
+
+$$
+S=\alpha(v_p-\lim v_n)
+$$
+
+### Condição suficiente de convergência de uma série redutível
+
+Como a condição de convergência não é muito fácil de verificar, usa-se o seguinte:
+
+Se $u_n\to L\in\R$, então a série redutível
+
+$$
+\sum ^{\infty }_{n=p}\left[ \alpha ( u_{n} -u_{n+k})\right]
+$$
+
+é convergente e a sua soma é
+
+$$
+S=\alpha(v_p-k\cdot L)
+$$
+
+:::details[Exemplos]
+
+$$\sum ^{\infty }_{n=2}\frac{2}{n^{2} +4n+3}$$
```

Comment:
```suggestion
$$
\sum ^{\infty }_{n=2}\frac{2}{n^{2} +4n+3}
$$
```

Otherwise `gatsby-remark-math` won't recognize this as a math block, but as inline math